src/components/modals/editMovie/editMovieModal.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import { shallowEqual, useDispatch, useSelector } from 'react-redux'
+import { ErrorMessage, Field, Form, Formik } from 'formik'
+
+import {
+  updateMovieRequested,
+  singleMovieFetchRequested,
+} from '../../../redux/actions'
+
+import { getMovie } from '../../../redux/selectors'
+import { editMovieValidationSchema } from './validationSchema'
+
+import styles from './editMovieModal.module.css'
+
+interface EditMovieModalProps {
+  id: string
+}
+
+export const EditMovieModal: React.FC<EditMovieModalProps> = ({ id }) => {
+  const dispatch = useDispatch()
+
+  React.useEffect(() => {
+    dispatch(singleMovieFetchRequested(id))
+  }, [])
+
+  const initialValues = {
+    id: 0,
```

Comment:
When a user clicks to edit a particular movie - it should be the id of that movie.